src/views/migrationpolicies/list/components/MigrationPolicyCreateForm/MigrationPolicyCreateForm.scss

```diff
@@ -0,0 +1,26 @@
+.migration-policy__form {
+  max-width: 50%;
+}
+
+.migration-policy__form-body {
+    margin: 1.5rem 0 0;
```

Comment:
What do you think if we use here patternfly variables like `--pf-global-spacer-md` etc.. ?

Reply:
see comment above :)

Reply:
I found this index for all global variables: https://www.patternfly.org/v4/developer-resources/global-css-variables/#component-variables
I switched all hard-coded values to PF vars, Thanks! :)